core/modules/views_ui/views_ui.admin.inc

```diff
@@ -799,6 +799,10 @@ function views_ui_add_form_save_submit($form, &$form_state) {
   $form_state['redirect'] = 'admin/structure/views';
   if (!empty($view->display['page'])) {
     $display = $view->display['page'];
+    backdrop_set_message(t('Your view was saved. Click here to <a href="@configure_view_link">configure the view</a> or <a href="@views_listing_page_link">return to the views listing page.</a>', array(
+      '@configure_view_link' => url('admin/structure/views/view/' . $view->name . '/configure'),
+      '@views_listing_page_link' => url('admin/structure/views/list'),
+    )));
```

Comment:
Could you move this down a few lines to just above

```
        $form_state['redirect'] = $one_path;  // PATH TO THE VIEW IF IT HAS ONE
```

Because there are situations where a page could be created but this redirect would not happen. Then you'd end up with two messages. This new one and the original `Your view was saved. You may configure it from the list below.`

It also wouldn't hurt to change `// PATH TO THE VIEW IF IT HAS ONE` to `// Path to the view if it has one.` to match our coding standards.